sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_pipeline_component_builder.py

```diff
@@ -278,6 +278,8 @@ def _map_internal_output_type(_meta):
                 meta=output_meta,
                 owner="pipeline",
                 description=self._args_description.get(key, None),
+                # store original node output to be able to trace back to inner node from a pipeline output builder.
+                node_output=value,
```

Comment:
is it possible to avoid maintain this here given we have already maintained an owner?

Reply:
For a `PipelineOutput`, it's owner will be the pipeline, but we want to track the inner node inside the pipeline here. Maybe I can rename this filed to make it clear

Reply:
Renamed to `binding_output`